exec/apk-gen.sh

```diff
@@ -11,7 +11,14 @@ then
 	git clone --quiet --branch=apk https://fossasia:$GITHUB_API_KEY@github.com/fossasia/susi_android apk > /dev/null
 	ls
 	cd apk
-	/bin/rm -f *
+	if [ "$CIRCLE_BRANCH" == "$PUBLISH_BRANCH" ]; then
+	    /bin/rm -f *
+	else
+	/bin/rm -f ../app/build/outputs/apk/fdroid/debug/app-fdroid-debug.apk
+	/bin/rm -f ../app/build/outputs/apk/playStore/debug/app-playStore-debug.apk
+	/bin/rm -f ../app/build/outputs/apk/playStore/debug/output.json
+	/bin/rm -f ../app/build/outputs/apk/playStore/debug/output.json
```

Comment:
You are removing generated apks, not the old ones

Reply:
@iamareebjamal how to delete the old ones ?

Reply:
Change the argument of rm to the new apks

I think you haven't quite understood the structure and process of how the apps are generated and the folder structure and content of the environment where the script is run